interferogram/aoi_merged_track_stitcher.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/env python3
+import sys
+import os
+import glob
+import shutil
+import json
+import math
+import numpy as np
+import re
+import hashlib
+from datetime import datetime
+
+from ifg_stitcher import IfgStitcher
+from utils.create_datasets import create_dataset_json
+from utils.createImage import createImage
+
+from osgeo import ogr, osr
+import matplotlib
+matplotlib.use('Agg')
+
+
+def get_version():
+    # dataset version: stored in conf/dataset_versions.json
+    ds_vers_cfg = os.path.normpath(
+        os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            '..', 'conf', 'dataset_versions.json'))
+    with open(ds_vers_cfg) as f:
+        ds_vers = json.load(f)
+    return ds_vers['S1-IFG-STITCHED']
```

Comment:
@DustinKLo the dataset shortname should be https://github.com/aria-jpl/ariamh/blob/standard-product/conf/dataset_versions.json#L8 however that only exists in the `standard-product` branch. I think we need @mkarim2017 to create a pull request first to merge in `standard-product` into `develop` then we can process this pull request. For now, fix this here and we'll need to sort out the `standard-product` vs `develop` branch when @mkarim2017 gets back.